.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space before `db.sqlite3`. It is recommended to remove this space to ensure the file is correctly ignored by Git.